packages/assets-controllers/src/NftController.ts

```diff
@@ -238,15 +242,15 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
     contractAddress: string;
     tokenId: string;
   }) {
-    return `${OPENSEA_PROXY_URL}/asset/${contractAddress}/${tokenId}`;
+    return `${OPENSEA_PROXY_URL}/chain/ethereum/contract/${contractAddress}/nfts/${tokenId}`;
```

Comment:
What do you think about extracting `ethereum` to a constant at the top of the file? It wasn't clear to me that we're using it to specify mainnet as the chain until I looked at the API docs.

Reply:
Will make it an enum.  v1 only supported mainnet, so will keep it that way.  But we'll support multichain soon.